src/main/java/miku/task/Task.java

```diff
@@ -0,0 +1,76 @@
+package miku.task;
+
+/**
+ * Represents a Task, which is a abstract parent class of all other task types.
+ */
+public abstract class Task {
+    private String desc;
+    private Boolean isDone = false;
+    private String done = " ";
+
+    public Task(String desc) {
+        this.desc = desc;
+    }
+
+    /**
+     * Instantiates a task object.
+     *
+     * @param desc
+     * @param isDone
+     */
+    public Task(String desc, boolean isDone) {
+        this.desc = desc;
+        this.isDoneInit(isDone);
+    }
+
+    /**
+     * If the value is read from the txt file, change the value accordingly
+     *
+     * @param isDone
+     */
+    public void isDoneInit(boolean isDone) {
```

Comment:
Maybe a name like initIsDone wuld be more appropriate as a name to following the coding standard?